org.civicrm.reqangular/src/common/angular-date/src/filters/CustomDateFilter.js

```diff
@@ -18,8 +20,18 @@ module.exports = function ($filter, DateFactory) {
         var beginningOfEra = DateFactory.createDate(0);
         var notEmpty = !Date.isSame(beginningOfEra);
 
-        if(Date.isValid() && notEmpty) return Date.format('DD/MM/YYYY');
+        if (!dateFormat) {
+            DateFormatFactory.getDateFormat().then(function (result) {
```

Comment:
Again, you're coupling the filter to the service, which in turns forces you to make the filter stateful because the service is async. What has changed since the last discussion about what approach to take? Why not store the format in a constant and inject it into the filter?